test/options_test.dart

```diff
@@ -214,7 +214,7 @@ void main() {
 
     test('given all values', () {
       final result = base.copyWith(
-        fallbackNamespace: another.fallbackNamespace,
+        fallbackNamespaces: another.fallbackNamespaces,
         namespaceSeparator: another.namespaceSeparator,
         contextSeparator: another.contextSeparator,
         pluralSeparator: another.pluralSeparator,
```

Comment:
Could you add a test case for the fallbacks?

- setup with two namespaces, where the key only exists in the last one

Reply:
I added some to i18next_test.dart